src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/ResultSet.java

```diff
@@ -48,4 +50,8 @@ protected boolean matchesPattern(String string, String pattern) {
         Matcher matcher = p.matcher(string);
         return matcher.find();
     }
+
+    public String getCursor() {
```

Comment:
return null or cursor?

Reply:
Removed this method as part of refactoring.